### PR TITLE
Add lat and lng atts to image-map shortcode

### PR DESCRIPTION
**Pull request by AK-digital**
Hi there, 

I'd like to add my little contribution to this cool plugin : my changes will allow editor to use lat="" and lng="" to the image-map shortcode, redefining the center point on first load if atts are set (''0' if not.)

My first official contribution to the community. Hope it will be published =)

Cheers !

### CODE REVIEW

**Comment by AK-digital** [comment]
Limit input lat and lng to int

File: shortcodes/class.image-shortcode.php [hunk 1]
```diff
@@ -48,7 +48,7 @@ protected function getHTML($atts='', $content=null)
 var image_src = '<?php echo htmlspecialchars($source, ENT_QUOTES); ?>';
 var img = new Image();
 var zoom = <?php echo $zoom; ?>;
-var map = window.WPLeafletMapPlugin.createImageMap(options).setView([0, 0], zoom);
+var map = window.WPLeafletMapPlugin.createImageMap(options).setView([<?php echo (int)$lat ? $lat : '0' ?>, <?php echo (int)$lng ? $lng : '0'?>], zoom);
```
Comment (bozdoz) [inline shortcodes/class.image-shortcode.php:51]: could you make this a variable instead?

Also maybe use `x` and `y` instead?

```php
$x = empty($x) ? 0 : $x
```

Comment (AK-digital) [reply shortcodes/class.image-shortcode.php:51]: Hi @bozdoz, don't see how you want it to be. Could you write it in the context please ?

Comment (bozdoz) [reply shortcodes/class.image-shortcode.php]: updated. Using 'x' and 'y' instead of lat/lng.  How does it look?  If it's good by you, I will merge

**Pull request merged**